applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.h

```diff
@@ -38,26 +38,25 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) CoulombYieldSurface : public YieldSu
     CoulombYieldSurface();
     explicit CoulombYieldSurface(const Properties& rMaterialProperties);
 
-    [[nodiscard]] double       GetFrictionAngleInRadians() const;
-    [[nodiscard]] double       GetCohesion() const;
-    [[nodiscard]] double       GetDilatancyAngleInRadians() const;
-    [[nodiscard]] unsigned int GetMaxIterations() const;
-    [[nodiscard]] double       GetConvergenceTolerance() const;
-    [[nodiscard]] double       GetKappa() const;
-    void                       SetKappa(double kappa);
+    [[nodiscard]] double GetFrictionAngleInRadians() const;
+    [[nodiscard]] double GetCohesion() const;
+    [[nodiscard]] double GetDilatancyAngleInRadians() const;
+    [[nodiscard]] double GetKappa() const;
+    void                 SetKappa(double kappa);
 
     [[nodiscard]] double YieldFunctionValue(const Vector& rSigmaTau) const override;
     [[nodiscard]] Vector DerivativeOfFlowFunction(const Vector&) const override;
     [[nodiscard]] Vector DerivativeOfFlowFunction(const Vector&, CoulombAveragingType AveragingType) const;
 
-    [[nodiscard]] double CalculateApex();
-    [[nodiscard]] double CalculatePlasticMultiplier(const Vector& rSigmaTau, const Vector& rDerivativeOfFlowFunction);
-    [[nodiscard]] double CalculateEquivalentPlasticStrain(const Vector&        rSigmaTau,
-                                                          CoulombAveragingType AveragingType,
-                                                          double               lambda) const;
+    [[nodiscard]] double CalculateApex() const;
+    [[nodiscard]] double CalculatePlasticMultiplier(const Vector& rSigmaTau,
+                                                    const Vector& rDerivativeOfFlowFunction) const;
+    [[nodiscard]] double CalculateEquivalentPlasticStrainIncrement(const Vector& rSigmaTau,
+                                                                   CoulombAveragingType AveragingType) const;
 
 private:
     void InitializeKappaDependentFunctions();
+    void CheckMaterialProperties();
```

Comment:
As a result of my previous suggestion:
```suggestion
    void CheckMaterialProperties() const;
```